main.py

```diff
@@ -1,7 +1,47 @@
 import init_django_orm  # noqa: F401
-
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    genres_list = ["Western", "Action", "Dramma"]
+    for genre in genres_list:
+        Genre.objects.create(
+            name=genre
+        )
+    actors_list = (["George", "Klooney"],
+                   ["Kianu", "Reaves"],
+                   ["Scarlett", "Keegan"],
+                   ["Will", "Smith"],
+                   ["Jaden", "Smith"],
+                   ["Scarlett", "Johansson"])
+
+    for actor in actors_list:
```

Comment:
You can unpacking value
```suggestion
    for first_name, last_name in actors_list:
```